opentelemetry-api/src/opentelemetry/_metrics/__init__.py

```diff
@@ -49,7 +49,9 @@
     _ProxyObservableUpDownCounter,
     _ProxyUpDownCounter,
 )
-from opentelemetry.environment_variables import OTEL_PYTHON_METER_PROVIDER
+from opentelemetry.environment_variables import (
+    _OTEL_PYTHON_METER_PROVIDER as OTEL_PYTHON_METER_PROVIDER,
+)
```

Comment:
Why did you want them to be private? I’m just curious because when I see constants being used to describe string literals I always liked it when the constant name matches the string literal exactly. As in [this example I wrote](https://github.com/open-telemetry/opentelemetry-python-contrib/blob/671aea32f9bbf53ace776c17ef2a80424d730fbb/instrumentation/opentelemetry-instrumentation-aws-lambda/src/opentelemetry/instrumentation/aws_lambda/__init__.py#L97-L98) where we have

```python
_HANDLER = "_HANDLER"
_X_AMZN_TRACE_ID = "_X_AMZN_TRACE_ID"
```

but if we do

```python
_OTEL_PYTHON_METER_PROVIDER = "OTEL_PYTHON_METER_PROVIDER"
```

then there is a discrepancy between constant name & string literal.

Reply:
It's temporary until metrics is declared stable, just in case we want to change things up. See #2249

Reply:
They are already in a private module, is this really necessary?